### DISCUSSION TITLE
LoTW Station Location query

### DISCUSSION THREAD

**Discussion by M0THL** [magicbug/Cloudlog#2775] [Q&A] [answered]
Morning all,

In the docs (https://github.com/magicbug/Cloudlog/wiki/LoTW-Import-&-Export-Documentation) we are told that a TQ8 file is built based on data contained in the Station Location area.

Which field in the Station Location page (or the DB field) is used to determine this? Or is it multiple fields? I hope someone knows to save me working my way through the code that builds the TQ8 :) I'm just refreshing my workflow after being mostly QRT this year.

**Comment by HB9HIL** [comment, marked as answer]
The LoTW TQ8 Builder uses the following Database values from the datatable `station_profile`

- `station_profile_name`  is "Profile Name" in Station Locations
- `station_callsign` is "Station Callsign" in Station Locations
- `station_dxcc` is the "Station DXCC" in Station Locations but as DXCC number

Make sure you use the latest version of Cloudlog.

**Reply by M0THL** [reply]
Thanks, that makes sense!

Yes, always running latest version.

Cheers

**Reply by HB9HIL** [reply]
You're welcome :)
Mark this discussion as answered please
